plonky2x/src/frontend/vars/byte.rs

```diff
@@ -299,11 +310,12 @@ mod tests {
         let mut builder = CircuitBuilder::<F, D>::new();
 
         let value = rand::random::<u8>();
-        let byte = builder.constant(value);
-        let nibbles = byte.to_nibbles(&mut builder);
+        let byte = builder.constant::<ByteVariable>(value);
+        let _nibbles = byte.to_nibbles(&mut builder);
 
         // TODO: test that the nibbles are correct
```

Comment:
This is awesome! Let's just add a test here, I can show you how to do that!

Reply:
oh, just saw the this, i added a weird test...